### PR TITLE
[Code Coverage] Improve coverage of fs.js

### PR DESCRIPTION
**Pull request by DanielBallaSZTE**
IoT.js-DCO-1.0-Signed-off-by: Daniel Balla dballa@inf.u-szeged.hu

### CODE REVIEW

File: test/run_pass/test_fs_exists.js [hunk 1]
```diff
@@ -25,6 +25,18 @@ var assert = require('assert');
 }
 
 {
+  var filePath = "";
+
+  fs.exists(filePath);
```
Comment (daeyeon) [inline test/run_pass/test_fs_exists.js]: The test case isn't completed with the comparison between expectation and result using assert. This problem can be found in other files as well as here.

**Comment by DanielBallaSZTE** [comment]
@daeyeon I've changed those two cases to check with assert, I didn't think it would need assertion, since the else case of both ifs are undefined.

**Comment by daeyeon** [comment]
@DanielBallaSZTE The others mentioned for same reason were not addressed. I added my suggestion in detail.

**Comment by DanielBallaSZTE** [comment]
@daeyeon Thanks for the suggestion, I'm going to keep those in mind next time. I hope everything is fixed now.

**Comment by daeyeon** [comment]
LGTM

**Comment by yichoi** [comment]
LGTM

**Pull request merged**